### PR TITLE
SWATCH-1851 Add swatch-product-configuration for RHEL ELS

### PR DESCRIPTION
**Pull request by kartikshahc**
Jira issue: [SWATCH-1851](https://issues.redhat.com/browse/SWATCH-1851)
Jira issue: [SWATCH-1839](https://issues.redhat.com/browse/SWATCH-1839)

### Setup

1.
1.

### Steps

1.
1.

### Verification

1.
1.

### CODE REVIEW

File: src/test/java/org/candlepin/subscriptions/tally/MetricUsageCollectorTest.java [hunk 1]
```diff
@@ -68,7 +68,7 @@
 class MetricUsageCollectorTest {
 
   public static final String RHEL_ENG_ID = "69";
-  public static final String RHEL_EUS_PAG_ENG_ID = "70";
+  public static final String RHEL_ELS_PAG_ENG_ID = "204";
```
Comment (lindseyburnett) [inline src/test/java/org/candlepin/subscriptions/tally/MetricUsageCollectorTest.java:71]: Looks like we were missing a "Y" here
`RHEL_ELS_PAG_ENG_ID` -> `RHEL_ELS_PAYG_ENG_ID`

**Comment by barnabycourt** [comment]
I believe the ELS change should only be made for the payg config file. The non paygo measurement can stay as EUS.

**Comment by kartikshahc** [comment]
> I believe the ELS change should only be made for the payg config file. The non paygo measurement can stay as EUS.

Sure. I think we can add ELS and EUS for non paygo. I don't see a problem in it.

File: swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_els_payg.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ contractEnabled: false #There is no need to generate an AWS contract listing bec
 
 metrics:
   - id: vCPUs
-    awsDimension: Centos_Test # Change the value to vCPU_Hour in SWATCH-1839
+    awsDimension: vCPU_Hour
```
[the remark this answers is not in the document]
Comment (kartikshahc) [reply swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_els_payg.yaml]: Done

**Review by ntkathole — APPROVED** [review]
Looks good to me. 
May be wait for merge till we get tally run for cost-mgmt events for today.

**Pull request merged**